src/main/java/piano/instruments/Instrument.java

```diff
@@ -0,0 +1,19 @@
+package piano.instruments;
+
+public class Instrument {
+    private String name;
+    private int sound;
+
+    public Instrument(String instrumentName, int instrumentSound) {
+        this.name = instrumentName;
+        this.sound = instrumentSound;
+    }
+
+    public Integer getInstrumentSound() {
```

Comment:
Consistency is important. 

If you are going to change the name (or type) of something one place you should change it everywhere.